app/controllers/tracks_controller.rb

```diff
@@ -14,8 +13,13 @@ def authenticated_index
     @tracks_data = SerializeTracks.(tracks, current_user)
   end
 
-  def authenticated_show
+  def show
     @user_track = UserTrack.for(current_user, @track)
+    render "tracks/show/joined"
```

Comment:
This might be right, but for a _reader_ it's very confusing. Shouldn't this be _inside_ the `if`?

Reply:
I was also slightly confused. I assume the `if` applied to the previous `render`?

Reply:
So confused. I've literally changed this 3 times to be correct and it seems to keep reverting somehow! I blame vim. Thanks both.